backend/src/main/java/com/cruru/advice/ConflictException.java

```diff
@@ -1,8 +1,10 @@
 package com.cruru.advice;
 
-public class ConflictException extends RuntimeException {
+public class ConflictException extends CruruCustomException {
+
+    private static final String STATUS_CODE = "409";
 
     public ConflictException(String message) {
-        super(message);
+        super(message, STATUS_CODE);
```

Comment:
HttpStatus로 바뀐다면, 이런 식으로 가독성이 좋아질 것 같네요.
```suggestion
        super(message, HttpStatus.CONFLICT);
```